train_configs/debug_model.toml

```diff
@@ -9,6 +9,7 @@ use_for_integration_test = true
 enable_profiling = true
 save_traces_folder = "profile_trace"
 profile_freq = 10
+enable_memory_snapshot = true
```

Comment:
nit: let's put the folder here as well to be consistent and informative

Reply:
added `save_memory_snapshot_folder` in .toml